USSF-ORBIT/ussf-portal-cms
# chore: Migrate existing cms data to postgres

## Description 
This PR migrates existing data from embedded Keystone to our standalone instance.

* Uses a dump of the embedded Keystone sqlite db to pull out data
* Creates a prisma migration to add the data to the postgres db

Fixes #sc-291

## Review Notes

Testing
* Run `npx prisma migrate dev` and make sure migration successfully completes
* Run portal and log in as cmsuser or cmsadmin
* Run `yarn dev` and open Keystone at localhost:3001
   * Validate there are 310 new bookmarks
   * Validate there are 14 new collections
